src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdAuthenticator.java

```diff
@@ -1,15 +1,15 @@
-package uk.gov.hmcts.reform.bulkscan.orchestrator.services;
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
 
 import uk.gov.hmcts.reform.idam.client.models.UserDetails;
 
 import java.util.function.Supplier;
 
-public class Authenticator {
+public class CcdAuthenticator {
```

Comment:
the way i see it is used in ccd client only as a container of tokens/ids. once imported in the class and used in whichever place a reviewer is aware this is ccd related. in case we re-use this for something else - both authenticator and authenticator factory can be moved out. oh, and authenticator factory was already prefixed with ccd. so just picked up one change to align both classes